src/test/java/io/gingersnapproject/testcontainers/annotation/WithPostgres.java

```diff
@@ -0,0 +1,19 @@
+package io.gingersnapproject.testcontainers.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import io.gingersnapproject.testcontainers.MySQLResources;
+import io.gingersnapproject.testcontainers.PostgresResources;
+
+import io.quarkus.test.common.QuarkusTestResource;
+
+@QuarkusTestResource(value = PostgresResources.class, restrictToAnnotatedClass = true, parallel = true)
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.TYPE)
+public @interface WithPostgres {
```

Comment:
This annotation looks very similar to the `WithMySQL` one, and in particular it does not contain nothing specific to the container type.
I don't know if it is possible to have only one of it, applying it to different resources, e.g.:
> @QuarkusTestResource(value = BaseGingersnapResourceLifecycleManager.class,

Reply:
Really nice, originally `BaseGingersnapResourceLifecycleManager` was abstract, so that wouldn't be possible, but I did some bigger changes now.